### DISCUSSION TITLE
.squid file association

### DISCUSSION THREAD

**Discussion by TPestaj** [CIRDLES/Squid#742] [Q&A] [answered]
Does anyone know how to associate Squid3 to open .squid files by default in Windows 10 (ie so it opens with a double-click) ?  Is it even possible?  It does not show up as an option for me when I try to choose it from the File Association dialog.  I hope I'm just missing something simple.

Thanks.
Tom

**Comment by bowring** [comment, marked as answer]
Great question!

Nope - windows is using its extension association already to open the .jar file.

BUT!

Remember Squid3 supports drag 'n drop of .squid files.

So, after doing the following, you can double click your .squid file and it will open Squid3.  
Then you need to drag the .squid file onto Squid3.

1) create a .bat file such as "RunSquid.bat"
2) add one line of text to it: "start C:\DEVELOPMENT\Squid3-1.10.6.jar" - using your location of Squid
3) double click your .squid file and follow the instructions to find an app ... choose RunSquid.bat

Remember Squid3 supports drag 'n drop

**Reply by TPestaj** [reply]
Ok, that's a helpful tip.  I had forgotten about the D 'n D functionality so I think I can make do with your .bat file suggestion.

Thanks!

**Comment by bowring** [comment]
@TPestaj - If you make an issue for this, I can update Squid3 so you don't have to drag n drop in the above scenario.
